File: src/main/kotlin/com/geonyeong/insurance/entity/TipEntity.java
package com.geonyeong.insurance.entity;


import com.geonyeong.insurance.dto.GuideIdDto;
import com.geonyeong.insurance.dto.TipIdDto;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Table(name = "tip")
@IdClass(TipIdDto.class)
@Getter
@Setter
public class TipEntity {
    @Id
    private @Column(name = "guide_id") Long guideId;
    @Id
    private @Column(name = "no") Long no;
    private @Column(name = "content") String content;
}

File: src/main/kotlin/com/geonyeong/insurance/entity/TipEntity.kt
package com.geonyeong.insurance.entity

import com.geonyeong.insurance.dto.TipIdDto
import javax.persistence.*

@Entity
@Table(name = "tip")
@IdClass(TipIdDto::class)
data class TipEntity(
        @Id
        @Column(name = "guide_id")
        var guideId: Long,
        @Id
        @Column(name = "no")
        var no: Long,
        @Column
        var content: String
){
        constructor():this(0,0,"")
}

